packages/math/src/IPoint.ts

```diff
@@ -35,6 +36,13 @@ export interface IPoint
  * @param {number} [y=x] - position of the point on the y axis
  */
 
+/**
+ * Creates a clone of this point
+ * @method clone
+ * @memberof PIXI.IPoint#
+ * @return {PIXI.Point} a copy of the point
```

Comment:
Should the return value be IPoint?

Reply:
Hi @bigtimebuddy, you're correct. I've just pushed up a fix 👍